cluster/module/server.c

```diff
@@ -1260,9 +1267,25 @@ restart_srv_from_file(gchar *path)
 			GRID_ERROR("Failed to unmarshall service info: (%d) %s",
 				err->code, err->message);
 		} else {
+
 			for (GSList *si = si_l; si; si = si->next){
 				struct service_info_s *si_data = si->data;
+				time_t time = get_file_timestamp(path);
+				score_t new_score = {SCORE_UNLOCK, time};
```

Comment:
`time` is the name of a function in `time.h`. Overriding this name should be avoided.